test_suite/tests/test_macros.rs

```diff
@@ -751,6 +751,31 @@ fn test_adjacently_tagged_enum() {
         ],
     );
 
+    // unit with excess content (f, g, h)
+    assert_de_tokens(
```

Comment:
Please add a test that checks the message when an unknown field is denied. See the tests in this file that use `assert_de_tokens_error`.